.github/workflows/create-release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     # Current latest (0.14.0) has a runtime error
     - name: Install Pack
       run: |
```

Comment:
fwiw you could skip this container create/copy/remove dance by using the linux install oneliner from the [buildpacks docs](https://buildpacks.io/docs/tools/pack/)
```
curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.15.0/pack-v0.15.0-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack
```

Reply:
As we're moving back to the`:latest` tag for pack, seems that this no longer applies? We do this in [some other places](https://github.com/paketo-buildpacks/github-config/blob/16dce0911daefb60de3f82e751ed052af850e9a7/language-family/scripts/.util/tools.sh#L101) in our tooling, but it introduces the problem of remembering to bump the pack version.